framework/platform/bpl/uci/db/bpl_db_uci.cpp

```diff
@@ -0,0 +1,93 @@
+#include "bpl_db_uci.h"
+
+
+namespace beerocks {
+namespace bpl {
+namespace db {
+
+bool uci_entry_exists(const std::string &config_file, const std::string &entry_type,
+                      const std::string &entry_name)
```

Comment:
(nitpick) The [proper terminology](https://openwrt.org/docs/guide-user/base-system/uci#uci_dataobject_model) is
```suggestion
bool uci_section_exists(const std::string &config, const std::string &section_type,
                      const std::string &section_name)
```
At least, I think that is what you mean with an "entry"... You may also mean "option" instead of "section".

Oh, and to make things confusing, sometime "package" is used instead of "config" :-) Maybe config_file isn't so bad after all...

Reply:
Renamed to match UCI names